src/components/uri/Visualization.tsx

```diff
@@ -368,16 +346,32 @@ export class VisualizationWrapped extends React.Component<
         switch (type) {
             case VisualizationTypes.TABLE:
                 return <TableComponent {...commonProps} totals={totals} />;
-            case VisualizationTypes.PIVOT_TABLE:
-                return <PivotTableComponent {...commonProps} totals={totals} />;
+            case VisualizationTypes.PIVOT_TABLE: {
+                const pivotBucketProps = mdObjectToPivotBucketProps(mdObject, filtersFromProps);
+                const pivotTableProps: IPivotTableProps = {
```

Comment:
This is just `commonProps` without `resultSpec`, right? Maybe you could simplify that?

Reply:
updated